CRM/Core/BAO/CustomField.php

```diff
@@ -738,7 +738,6 @@ public static function addQuickFormElement(
       }
 
       // Add data for popup link. Normally this is handled by CRM_Core_Form->addSelect
-      $isSupportedWidget = in_array($widget, ['Select', 'Radio']);
```

Comment:
Confirmed doing a grep this is unneeded

Reply:
https://github.com/civicrm/civicrm-core/pull/16493